exampleSite/hugo.yaml

```diff
@@ -174,6 +174,11 @@ params:
         # matomo:
         #   instance: matomo.example.com
         #   siteId: 1 # Or any other number
+        # # Umami
+        # umami:
+        #   proto: https
```

Comment:
Let's rename `proto` to `scheme`.

Reply:
Agree. A lot better than mine naming :)
I updated also the guides PR.